monitoring/templates/alertmanager.yml

```diff
@@ -0,0 +1,30 @@
+global:
+  slack_api_url: ${slack_url}
+
+route:
+  receiver: 'slack'
+  routes:
+    - receiver: slack
+      match:
+        severity: warning
+    - receiver: opsgenie
+      match:
+        severity: critical
+    - receiver: opsgenie
+      match:
+        alertname: DeadMansSwitch
```

Comment:
Is the DeadMansSwitch tested? On k8s we use a small proxy to send it as a heartbeat to opsgenie, but I'm not sure anymore why we did that.

Reply:
Ok, probably won't work, but that can be added in a subsequent PR.